src/mango/src/mango_idx_view.erl

```diff
@@ -120,7 +120,7 @@ is_usable(Idx, Selector, SortFields) ->
     % and the selector is not a text search (so requires a text index)
     RequiredFields = columns(Idx),
 
-    % sort fields are required to exist in the results so 
+    % sort fields are required to exist in the results so
```

Comment:
unrelated whitespace change

Reply:
This was my editor removing whitespaces.